src/constants.js

```diff
@@ -9,7 +9,7 @@ const TYPES = [
   "ci",
 ];
 
-const SCOPES = ["API", "core", "editor"];
+const SCOPES = ["API", "core", "editor", "benchmark"];
```

Comment:
1. What is the criteria for adding a new scope instead of using `test` or `ci`? Existing scopes are broad while `benchmark` seems granular.
2. If we decide to add a new scope, can [we add a test](https://github.com/n8n-io/validate-n8n-pull-request-title/blob/master/src/validatePrTitle.test.js)?

Reply:
`test` and `ci` define the type of the commit, scope defined the component it is for. So we could have e.g. `ci(benchmark)`. We could also consider the benchmarks to be just tests for the `cli` (i.e. always `test(core)`) or not use a scope at all. It is its own package and application, so I think it would make sense to have a separate scope for it.